js/hoot/control/utilities/Folder.js

```diff
@@ -204,18 +230,31 @@ Hoot.control.utilities.folder = function(context) {
                   } while(Math.abs(_size) >= 1000 && u < units.length - 1);
                   return _size.toFixed(1)+' '+units[u];
             });
-
+              // add date added  & last accessed date for each dataset
               if(container.attr('id')==='datasettable'){
                  nodeEnter.filter(function(d){return d.type==='dataset';}).append('text')
                       .style('fill',fontColor)
                       .attr('dy',3.5)
-                    .attr('dx',function(){
-                        return '75%';
-                    })
-                    .attr('text-anchor','end')
-                    .text(function(d) {
-                        return d.date;
-                    });
+                      .attr('dx',function(){ return '80%';})
+                      .attr('text-anchor','end')
+                      .text(function(d) {
+                          return d.date;
+                      });
+                  nodeEnter.filter(function(d) { return d.type ==='dataset';})
+                      .append('text')
+                      .style('fill', fontColor)
+                      .attr('dy', 3.5)
+                      .attr('dx', function() { return '50%'; })
+                      .attr('text.anchor', 'end')
+                      .text(function(d) {
+                            var lastAccessed = d.lastAccessed,
+                                timeAgo = lastAccessed.replace(/[-:]/g, '');
+                            var dateActive = moment(timeAgo).fromNow();
+                            // show warning if dataset hasn't been accessed in 2 months
+                            var oldData = dateActive.split(' ')[1] === 'months' && dateActive.split(' ')[0] >= 2;
```

Comment:
This isn't very robust.  What is there is data last accessed "a year ago".  Let's not use the fromNow() value for this conditional, let's use something like `moment.now().diff(moment(timeAgo), 'days') > 60`

Reply:
I was building this under the assumption we wouldn't have any datasets that were >1 year old (based on introducing lastAccessed and auto-deleting datasets that are 3 months or older). Now that we are not auto deleting though, you are right.